templates/admin/job_template/index.html.ep

```diff
@@ -108,9 +112,8 @@
             <div class="form-group row">
                 <label for="editor-template" class="col-sm-5 control-label editor-yaml-guide" style="display: none;">
                     <div>
-                        <div>Define job templates per arch/medium:</div>
-                        <code>
-defaults:
+                        <div>Basic YAML structure for defining job templates:</div>
```

Comment:
I don't think "YAML structure" is a good word to ask users for. IMHO we want to avoid this rather technical term which of course is the format we use but write from the users point of view. I liked the former better

Reply:
I think the user actually needs to deal with the fact that this is indeed YAML. I'd like to say that this example gives the basic structure. So it is clear that this is the most generic of the examples. The "per arch/medium" is redundant because it is clear from the structure itself.

Reply:
I kinda agree that we need users to be aware of what they're working with. And it is less redundant indeed.